pandas/core/groupby/groupby.py

```diff
@@ -581,6 +581,16 @@ def wrapper(*args, **kwargs):
             if 'axis' not in kwargs_with_axis or \
                kwargs_with_axis['axis'] is None:
                 kwargs_with_axis['axis'] = self.axis
+            if name == 'hist' and kwargs.get('equal_bins', False):
```

Comment:
Can just do `kwargs.get('equal_bins')` - no need for the `False` argument

Reply:
For some reason py3 and py2 `kwargs` behavior are different. In py3 `kwargs` defaults are not set here if the parameter is not provided by the user. I opened an issue on this (#22285). I'm not sure where to move the code (per @jreback's comment) to avoid this.